project/plugins.sbt

```diff
@@ -11,4 +11,5 @@ resolvers += Resolver.url(
   new URL("https://broadinstitute.jfrog.io/broadinstitute/libs-release/")
 )(publishPatterns)
 
-addSbtPlugin("org.broadinstitute.monster" % "sbt-plugins-jade" % "0.6.0")
+addSbtPlugin("org.broadinstitute.monster" % "sbt-plugins-jade" % "0.7.0")
+addSbtPlugin("org.broadinstitute.monster" % "sbt-plugins-scio" % "0.7.0")
```

Comment:
Nit: You could make a `val` to store the version string, to make upgrades slightly easier in the future